networking/amplify/backend/api/mobile/custom-roles.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
this role is required to ensure that the `createGame` Lambda function is explicitly provided with its Execution Role